amplify.yml

```
@@ -0,0 +1,29 @@
+  cache:
+    paths:
+      - ch11-frontend/node_modules/**/*
+
+backend:
+  phases:
+    preBuild:
+      commands:
+        - echo "Backend deployed via ECS Fargate"
+        - echo "API endpoint: https://api.ch11.ai"
+
+# Environment variables for React app
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)